Sources/PackageManagerDocs/Documentation.docc/SwiftSDKCommands.md

```diff
@@ -9,9 +9,15 @@ Perform operations on Swift SDKs.
 
 ## Overview
 
-Overview of package manager commands here...
+By default, Swift Package Manager compiles code for the host platform on which you run it.
+Swift 6.1 introduced SDKs to support cross-compilation.
 
-<!-- reference content for the CLI commands `swift package ...` -->
+SDKs are tightly coupled with the toolchain used to create them.
+Supported SDKs are distributed by the Swift project with links on the [installation page](https://www.swift.org/install/) for macOS and Linux, and included in the distribution for Windows.
+
+Additionally, the Swift project provides a tooling repository called [swift-sdk-generator](https://github.com/swiftlang/swift-sdk-generator) that you can use to create a custom SDK for your preferred platform.
+
+For more information on the introduction of Swift SDKs, refer to the Swift evolution proposal [SE-0387](https://github.com/swiftlang/swift-evolution/blob/main/proposals/0387-cross-compilation-destinations.md).
```

Comment:
I'm not sure linking to SE proposals as documentation is the best idea as a form of documentation, since they can become stale over time or be lacking in details we want to communicate. This seems like a good addition for now, although as a follow-on I think we should probably say something more like "was introduced in SE-0387" (with the link), but then go on to repeat the relevant parts here in detail along with any additional information we want to communicate.